### DISCUSSION TITLE
Media queries stopped working - perhaps Tailwind issue?

### DISCUSSION THREAD

**Discussion by AshSimmonds** [saadeghi/daisyui#908] [Q&A]
Not sure when or why, but pages now always resolve as mobile, I'd be able to fix it but I simply don't know where to look.

Eg: https://bluedwarf.space/mission/ (built on Astro)

The pips at the ~~top~~ bottom of the page should indicate what size the page is, and render depending on that.

![s6cHdhDKJM](https://user-images.githubusercontent.com/9034041/176824314-3364fb5f-1cd0-4878-bead-b6ba906e9d99.png)

Any ideas?

**Comment by saadeghi** [comment]
Hey. I see that you closed your issue here  
https://github.com/saadeghi/daisyui/issues/902#event-6889502075

What was the problem?

**Reply by AshSimmonds** [reply]
After I opened it I realised it was better as a `discussion` topic rather than an `issue`, as I don't know if DaisyUI is the source yet.

**Comment by AshSimmonds** [comment]
Oh, I moved the media query identifier pips to the bottom of the page just above the footer. When you resize the window they should reflect page size.
